CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## [Unreleased]
 
+### Added
+
+-
+
+### Fixed
+
+- `get_mlflow_config` now uses the kedro context config_loader to get configs (#66 #64 #54 #30 #31 #72)
+
+### Changed
+
+- `MlflowNodeHook` have now a before_pipeline_run hook allowing to load the kedro context, get configs and populating MlflowNodeHook attributes
+- `mlflow init` is now getting conf path from `context.CONF_ROOT`
```

Comment:
This goes to "fixed" I think.  

"- `mlflow init` is now getting conf path from `context.CONF_ROOT` instead of hardcoded conf folder. This makes the package robust to kedro changes."

Reply:
We didn't raise an issue for that. But yeah, it sound like a fix

Reply:
> We didn't raise an issue for that
Because no one has spotted the error before :)